frontend/js/src/brainzplayer/BrainzPlayer.tsx

```diff
@@ -118,6 +118,14 @@ function isListenFromDatasource(
   return undefined;
 }
 
+export function ToastMsg({ title, message }: ToastMsgProps) {
+  return (
+    <div>
+      <b>{title}</b>
+      <p>{message}</p>
+    </div>
+  );
+}
```

Comment:
This isn't the place to export this component.
We are eventually going to use it across the codebase, so we want to import it from a more "neutral" place.

Maybe its own component/file in the `js/src/notifications` folder?

Reply:
Sounds good to me :+1: